activerecord/test/cases/quoting_test.rb

```diff
@@ -71,9 +71,33 @@ def test_quoted_time_utc
         with_timezone_config default: :utc do
           t = Time.now.change(usec: 0)
 
-          expected = t.getutc.change(year: 2000, month: 1, day: 1)
-          expected = expected.to_s(:db).sub("2000-01-01 ", "")
+          puts t
 
+          expected = t.change(year: 2000, month: 1, day: 1)
+          expected = expected.getutc.to_s(:db).slice(11..-1)
+
+          assert_equal expected, @quoter.quoted_time(t)
+        end
+      end
+
+      def test_quoted_time_utc_dst
+        with_timezone_config default: :utc do
+          t = Time.new(2000, 7, 1)
```

Comment:
This date was chosen as per the examples in Ruby's documentation for [`.dst?`](https://ruby-doc.org/core-2.2.0/Time.html#method-i-dst-3F)